database/migrations/000008_create_transaction_outputs_table.up.sql

```diff
@@ -1,18 +1,19 @@
-CREATE TABLE `transaction_outputs`
+CREATE TABLE transaction_outputs
 (
-    `id`             BIGINT UNSIGNED NOT NULL AUTO_INCREMENT,
-    `transaction_id` BIGINT UNSIGNED NOT NULL,
-    `index`          INT UNSIGNED    NOT NULL,
-    `value`          BIGINT UNSIGNED NOT NULL,
-    `script_pub_key` BLOB            NOT NULL,
-    `is_spent`       TINYINT         NOT NULL,
-    `address_id`     BIGINT UNSIGNED NULL,
-    PRIMARY KEY (`id`),
-    INDEX `idx_transaction_outputs_transaction_id` (`transaction_id`),
-    CONSTRAINT `fk_transaction_outputs_transaction_id`
-        FOREIGN KEY (`transaction_id`)
-            REFERENCES `transactions` (`id`),
-    CONSTRAINT `fk_transaction_outputs_address_id`
-        FOREIGN KEY (`address_id`)
-            REFERENCES `addresses` (`id`)
+    id             BIGSERIAL,
+    transaction_id BIGINT NOT NULL,
+    index          BIGINT CHECK (index >= 0) NOT NULL,
```

Comment:
here too, index is uint32, originally, so needs to make sure values stay within uint32 range